sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -1162,7 +1162,7 @@ export class BlobServiceClient extends StorageClient {
     // AsyncIterableIterator to iterate over containers
     const listSegmentOptions: ServiceListContainersSegmentOptions = {
       ...options,
-      include
```

Comment:
What's changed which caused this fix?

Reply:
Touched this when adding support for `includeDeleted`.